src/index.js

```diff
@@ -0,0 +1,56 @@
+import React, {Component} from 'react';
+import ReactDOM from 'react-dom';
+import "./css/bootstrap.min.css";
+import "./css/my-styles.css";
+
+const name = "World";
+
+
+// 1. -	React.createElement
+const el1 = React.createElement("h1",
+    {className: "title"},
+    "This is a component, created by React.createElement"
+);
+
+// 2. -	functional component
+var el2 =  (name) => (<p>Hello {name}! - functional component</p>);
+
+// 3. -	React.Component
+
+class El3 extends Component {
+    constructor() {
+        super();
+        this.state = {
+            value: "Click to change my name",
+            buffer: ""
+        };
+        this.setName = this.setName.bind(this);
+        this.setBuffer = this.setBuffer.bind(this);
+    }
+    setBuffer(e){
```

Comment:
You can define the function as arrow function `setBuffer = () => {}`. So you won't need `.bind` in constructor

Reply:
Rewrited